hail/src/main/scala/is/hail/expr/ir/Compile.scala

```diff
@@ -124,6 +124,18 @@ object Compile {
     apply[AsmFunction5[Region, T0, Boolean, T1, Boolean, R], R](ctx, print, FastSeq((name0, typ0, classTag[T0]), (name1, typ1, classTag[T1])), body, optimize = true)
   }
 
+  def apply[T0: ClassTag, T1: ClassTag, R: TypeInfo : ClassTag](
```

Comment:
where is this used?

Reply:
In testutils. JvmCompileUnoptimized path. Currently only enabled for testGroupByKey

Reply:
can you point me to the line?

Reply:
https://github.com/hail-is/hail/pull/8124#discussion_r381999923

Reply:
I double checked that a constructor didn’t already exist. The combination of printer and optimize with 3 Long’s didn’t

Reply:
I can't see where you actually call compile in a changed line of code, though.